src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,11 +1,43 @@
 package com.google.sps.data;
 
-public class Comment{
+public final class Comment{
     private final String name;
     private final String comment;
+    private final long id;
 
-    public Comment(String name, String comment){
+    public Comment(String name, String comment, long id){
         this.name = name;
         this.comment = comment;
+        this.id = id; 
+    }
+
+    public String getDisplayName(){
+        return name; 
+    }
+
+    public String getComment(){
+        return comment; 
+    }
+
+    public long getID(){
+        return id; 
+    }
+
```

Comment:
need to implement hashCode when you implement equals. I believe nikita implemented in PR#3 so you can rebase on top of that when she merges.